templates/static-type-test-wrapper.twig

```diff
@@ -0,0 +1,3 @@
+$return = {{class}}::{{function.name}}{% include 'function.twig' with {'function': function} only %};
+
+{% include 'type-tests.twig' with {'return': function.return, 'variable': '$return'} only %}
```

Comment:
no new line at end of file, but it's a twig template so it doesn't matter, right?

Reply:
yes, would also mess up the output